test/esinstall/rollup/rollup.test.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const {install} = require('../../../esinstall/lib');
+
+describe('Rollup config', () => {
+  it('it can be passed through the rollup option', async () => {
+    const cwd = __dirname;
+    const dest = path.join(cwd, 'test-rollup');
+    const spec = 'svelte-routing';
+
+    const {
+      importMap: {imports}
+    } = await install([spec], {
+      cwd,
+      dest,
+      rollup: {
+        plugins: [require('rollup-plugin-svelte')()]
+      }
+    });
+
+    // install would have thrown without the plugin, so getting here is enough.
```

Comment:
👍🏻  Good comment. This helps me understand this test better

Reply:
I would think about if this is true, testing the inverse: test that it _does_ throw without the plugin in another test.

Reply:
I think this can probably be done quickly, I'll give it a shot.